pyproject.toml

```diff
@@ -22,6 +22,7 @@ openai = "^1.33.0"
 posthog = "^3.5.0"
 pytz = "^2024.1"
 sqlalchemy = "^2.0.31"
+google-generativeai = "^0.8.3"
```

Comment:
Hey make a optional dependency. You can check how we have done for other LLMs

Reply:
Done!